### PR TITLE
CB-10363: Cleaning up the 'plugin spec' documentation

### PR DESCRIPTION
**Pull request by omefire**
CB-10363

### CODE REVIEW

**Comment by omefire** [comment]
@rakatyal can you please review these changes ?

**Comment by rakatyal** [comment]
There is already a Jira (CB-10363) for this under Docs 6.x tag. Can you please refer to that one instead and close this one?

File: www/docs/en/dev/plugin_ref/spec.md [hunk 1]
```diff
@@ -98,12 +98,14 @@ Here's a list of the default engines that the `<engine>` tag supports:
 
 * `cordova`
 * `cordova-plugman`
-* `cordova-amazon-fireos`
 * `cordova-android`
 * `cordova-ios`
 * `cordova-blackberry10`
 * `cordova-wp8`
 * `cordova-windows8`
```
Comment (rakatyal) [inline www/docs/en/dev/plugin_ref/spec.md]: We can remove windows8 as well.

File: www/docs/en/dev/plugin_ref/spec.md [hunk 2]
```diff
@@ -345,17 +347,20 @@ associating the element's children with that platform.
 Platform names should be lowercase. Platform names, as arbitrarily
 chosen, are listed:
 
-* amazon-fireos
 * android
 * blackberry10
 * ios
 * wp8
 * windows8
```
Comment (rakatyal) [inline www/docs/en/dev/plugin_ref/spec.md]: Here too.

**Comment by rakatyal** [comment]
@omefire: Can you close this PR? I took care of most of these changes in my commit.

**Pull request closed without merging**